src/core/reactor.ts

```diff
@@ -1885,7 +1885,8 @@ export class App extends Reactor {
      */
     constructor(executionTimeout: TimeValue | undefined = undefined, 
                 keepAlive: boolean = false, 
-                fast: boolean = false, 
+                fast: boolean = false,
+                advanceMessageInterval: TimeValue | undefined = undefined,
```

Comment:
It's a bit ugly to require this in the constructor as it will typically be `undefined`. Wouldn't it be better to just have a class variable and a setter? We can then use the value at the time of scheduling (rather than use it as a parameter of the action).

Reply:
Thanks for the suggestion!